File: C++/Product.cpp
#include <iostream>
#include <string>
using namespace std;

class Product{

	private:
		int price;
		string idProduct;

	public:

		Product(int price, string idProduct){
			this->price = price;
			this->idProduct = idProduct;
		}

		Product(){
			this->price = 0;
			this->idProduct = "";
		}

		void setPrice(int price){
			this->price = price;
		}
		
		void setIdProduct(string idProduct){
			this->idProduct = idProduct;
		}


		int getPrice(){
			return this->price;
		}

		string getIdProduct(){
			return this->idProduct;
		}

		
		~Product(){
			
		}

};

File: PYTHON/Product.py
class Product:

	def __init__(self):
		self.price = 0
		self.idProduct = ""



	def setPrice(self, price):
		self.price = price

	def setIdProduct(self, idProduct):
		self.idProduct = idProduct


	def getPrice(self):
		return self.price

	def getIdProduct(self):
		return self.idProduct